addons/spreadsheet/static/src/global_filters/components/relative_date_value/relative_date_value.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="spreadsheet_edition.RelativeDateValue">
+        <div class="d-flex">
+            <select class="o-input o-gf-reference-selection" t-on-change="onReferenceChanged">
+                <option value="" t-att-selected="!props.value">No value</option>
+                <option t-foreach="relativeDateRangesReferences"
+                        t-as="reference"
+                        t-key="reference.type"
+                        t-att-value="reference.type"
+                        t-att-selected="reference.type === props.value?.reference"
+                        t-esc="reference.description"/>
+            </select>
+            <input t-att-disabled="!props.value || props.value.reference === 'this'"
+                   t-on-change="onIntervalChanged"
```

Comment:
increasing the number with the arrow keys automatically triggers "onIntervalChanged" but writing the number manually doesn't until we hit enter or blur

Reply:
I'm not sure the behavior we want... only on enter ? on blur ? on input ? on change ? Not sure...